### ISSUE TITLE
Max/min metric in table columns

### ISSUE THREAD

**Issue by depthwise** [comet-ml/issue-tracking#268]
Maybe there is such an option, but I couldn't find it. Here's what I'd like to have: an option to create a table such that it shows not just the latest value of a metric, but also the min/max value thus far in the experiment. I.e. if I'm training a segmentation model I care about the max mIoU and min loss when the experiment ends, not whatever those two metrics are at the end of training.

If this is already doable, please make the feature more discoverable, in part by commenting here.

Thanks!

**Comment by gidim** [comment]
Hi @depthwise  - you can achieve this by reporting it manually from your code.
for example if you have something like this today:

```
for i in (epochs):
   loss = compute_loss(model)
   experiment.log_metric("loss", loss)
```

just keep a running minimum and report it again


```
min_loss = sys.maxint
for i in (epochs):
   loss = compute_loss(model)
   min_loss = min(loss, min_loss)
   experiment.log_metrics({"loss": loss, "min_loss":min_loss})
```

We are adding native support in the UI for this as well
